spanner_orm/tests/model_test.py

```diff
@@ -19,12 +19,130 @@
 from unittest import mock
 
 from absl.testing import parameterized
+from spanner_orm import error
 from spanner_orm import field
 from spanner_orm.tests import models
 
 
 class ModelTest(parameterized.TestCase):
 
+  @mock.patch('spanner_orm.table_apis.find')
+  def test_find_calls_api(self, find):
+    mock_transaction = mock.Mock()
```

Comment:
Is there any auto spec that can be used here and below?

Reply:
No idea, though I totally agree that autospec is a good idea. I didn't write this code, I just moved it from one file to another. (This PR is 99% moving stuff around.)

Reply:
Gotcha, I didn't register that this was all in the test file that you deleted, but that makes sense.